### DISCUSSION TITLE
CSV and XML does not include field shownerid on export file

### DISCUSSION THREAD

**Discussion by LinoRiccio** [YetiForceCompany/YetiForceCRM#15759] [General]
Hello, after updating from version 6.2 to 6.3, I notice that the value of the "share with" fields is no longer included in the exported files. The module in question is a new module created by us. Is it possible to intervene on the configuration? A greeting

**Comment by rskrzypczak** [comment]
Unfortunately, there is a bug here that only occurs with quick export. (https://github.com/YetiForceCompany/YetiForceCRM/issues/15766)
This error does not occur in standard export.

**Reply by Blessings-Nthara** [reply]
@rskrzypczak thank you for your response, i would kindly like to ask how i can fix this bug?
